src/types/Person.tsx

```diff
@@ -0,0 +1,9 @@
+export interface Person {
+  'name': string,
```

Comment:
```suggestion
  name: string,
```
and the same way all the way down